Weather App/Common/Classes/CheckButton.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  CheckBox.swift
+//  Weather App
+//
+//  Created by Ivan Stajcer on 02.09.2021..
+//
+import UIKit
+
+class CheckButton : UIButton {
+
+    var isChecked: Bool! {
```

Comment:
Force unwrap? Is there a better way to handle this property?

Reply:
I removed it, force unwrap was not needed.